lanelet2/lanelet2_core/include/lanelet2_core/Attribute.h

```diff
@@ -5,9 +5,10 @@
 #include <boost/variant/variant.hpp>
```

Comment:
Why am I seeing more changes to lanelet2? We already merge those. Does you branch need to merge noetic/develop back in?

Reply:
I'm not sure why it did that. I pulled in the noetic/develop changes